README.md

```diff
@@ -10,9 +10,12 @@ It makes your competitive coding life easier :grin: by automating many things fo
 
 ---
 
+[![Open Collective](https://opencollective.com/cpeditor/tiers/support-us.svg?avatarHeight=60)](https://opencollective.com/cpeditor#section-contribute)
+<br>
```

Comment:
Why `<br>`?

Reply:
It wasn't breaking a line in Preview. When I added it worked.

Reply:
You can preview what looks better, with or without line break.